airflow/models.py

```diff
@@ -219,6 +219,7 @@ def __init__(
         self.file_last_changed = {}
         self.executor = executor
         self.import_errors = {}
+        self.has_logged = False
```

Comment:
Can you add a docstring for this param?

https://github.com/Noremac201/incubator-airflow/blob/b271cab470bfebc7e46aae0377f68dabe3e03c5f/airflow/models.py#L191:198

Reply:
has_logged isn't passed to the constructor, do you still want me to document it?

Reply:
I know but the param name itself isn't self-explanatory so I am saying so.